solvers/l_bfgs_b.py

```diff
@@ -28,13 +30,22 @@ class Solver(BaseSolver):
     ]
 
     def skip(self, X, y, lmbd, fit_intercept):
-        # XXX - not implemented but this should be quite easy
-        if fit_intercept:
-            return True, f"{self.name} does not handle fit_intercept"
+        # XXX - intercept not implemented for sparse X for now
+        if fit_intercept and issparse(X):
+            return True, \
```

Comment:
We try to avoid using `\`, you can wrap the return inside parenthesis to allow for linebreaks inside

Reply:
Handled at commit 2b634fcd7d297676fe9f00bf48360966ad813bef